### PR TITLE
MenuBar settings selection fix

### PR DESCRIPTION
**Pull request by blakegreen32**
Fixed MenuBar.js isSelected function to take into account settings menu selections

### CODE REVIEW

File: src/foam/nanos/menu/MenuBar.js [hunk 1]
```diff
@@ -84,6 +84,10 @@ foam.CLASS({
         return false;
       }
 
+      if ( this.selected === menu.id && this.selected.parent != 'settings' ) {
```
Comment (kgrgreer) [inline src/foam/nanos/menu/MenuBar.js]: Why is this required? Add comment to explain.

Comment (blakegreen32) [reply src/foam/nanos/menu/MenuBar.js]: I have actually changed it because this wasnt persisting in every case, I used window.location.hash.includes('#set') so that if the user navigates to any settings pages the previous menu items wont stay underlined. With the latest commit the menu items work as they should and only show as selected if the user is on that page and has not navigated to a settings page.

Comment (blakegreen32) [reply src/foam/nanos/menu/MenuBar.js]: I think this makes sense as if any new menu items are added to the settings menu they will include the hash of "#set" in them

**Review by kirkeaton — APPROVED** [review]
